### ISSUE TITLE
[Win64] real constants parsed as 0

### ISSUE THREAD

**Issue by Trass3r** [ldc-developers/ldc#761]
Lexer::inreal calls Port::strtold to parse the real constants in std.math.

``` d
enum real PI =         0x1.921fb54442d18469898cc51701b84p+1L;
```

This maps to the msvc strtold and just returns 0, also see #747.

**Comment by redstar** [comment]
The compiler part is hopefully fixed with commit 2fdd4fe502ad9b7c5d188cf4425857ecc300931d.

**Comment by kinke** [comment]
FWIW, although we only support VS 2015 anymore, this is fixed by #2054.
